packages/site/src/pages/index.tsx

```diff
@@ -1,11 +1,26 @@
 import * as React from 'react';
-import { ThemeProvider } from 'styled-components';
-import { Grid, Stack, Text, Spacer } from 'noya-designsystem';
-import { PagerView, View } from '../components/PagerView';
+import styled from 'styled-components';
+
+import {
+  Button,
+  Grid,
+  InputField,
+  Spacer,
+  Stack,
+  Text,
+  mediaQuery,
+  textStyles,
+} from '../system';
+import { GitHubIcon } from '../components/Icons';
+import { Layout } from '../components/Layout';
+import { Packages } from '../components/Packages';
+
 import logoSrc from '../assets/logo.svg';
-import * as theme from '../theme';
+import sketchSrc from '../assets/sketch.svg';
+import figmaSrc from '../assets/figma.svg';
+import xdSrc from '../assets/xd.svg';
 
-const preventTextOprhan = (text: string) => text.replace(/ /g, '\u00a0');
+const preventTextRunt = (text: string) => text.replace(/ /g, '\u00a0');
```

Comment:
Was using the wrong terminology 😇  more info [here](https://brennenreece.squarespace.com/blog/fixing-runts-in-indesign-using-grep) if you're curious.